src/post/post.repository.knex.ts

```diff
@@ -150,6 +150,12 @@ export class KnexPostRepository {
         });
     }
 
+    async getByUuid(uuid: Post['uuid']): Promise<Post | null> {
+        return await this.getByQuery((qb: Knex.QueryBuilder) => {
+            return qb.where('posts.uuid', uuid);
+        });
+    }
+
```

Comment:
This query will change when we'll have the new table that maps ghost-uuid to our AP posts